frontend/src/actions/transfers/transfer.ts

```diff
@@ -80,7 +81,9 @@ export class Transfer extends MultiStepAction implements Encodable<TransferData>
   readonly date: Date;
   private _requestInformation?: RequestInformation;
   private _expired: boolean;
+  private _claimCount: number;
   private _withdrawn: boolean;
+  private listenerCleanupCallbacks: Array<CallableFunction> = [];
```

Comment:
Wonder why this is implemented as an array.. There should never be more than one, right? And stuff can go wrong if for any reason a second listener is started. Would it make sense to just disallow another call to `startClaimEventListeners` by throwing an error if there is already a listener?